routes/campgrounds.js

```diff
@@ -0,0 +1,109 @@
+const express = require("express");
+const router = express.Router();
+const Campground = require("../models/campground");
+const Comment = require("../models/comment");
+const middleware = require("../middleware");
+
+router.get("/campgrounds", function (req, res) {
+  //get all campground from
+  Campground.find({}, function (err, allCampgrounds) {
+    if (err) {
+      console.log(err);
```

Comment:
Avoid console logs. Use a better logging feature. You can checkout winston from npm.